client/src/components/App/Content/LandingPage/adminLandingPage/UnusualCard/UnusualCardStyles.ts

```diff
@@ -0,0 +1,58 @@
+import { makeStyles } from '@material-ui/styles';
+
+export const cardWidth = '14vw';
+export const cardHeight = '38vh';
+
+const useStyles = makeStyles({
+    unusualCard: {
+        width: cardWidth,
+        height: cardHeight,
```

Comment:
I think dropping both width and height from here will be better (defining both using xs and md in the grid will help you control the component size better)